include/stl2/detail/iterator/increment.hpp

```diff
@@ -96,8 +97,8 @@ STL2_OPEN_NAMESPACE {
 		concept bool Decrementable =
 			Incrementable<I> &&
 			requires(I& i) {
-				{ --i } -> Same<I&>&&;
-				{ i-- } -> Same<I>&&;
+				{ --i } -> Same<I>&;
+				i--; Same<decltype(i--), I>;
```

Comment:
Missing `requires` here:
```suggestion
				i--; requires Same<I, decltype(i--)>;
```